### ISSUE TITLE
3rd step in the Procedure is not correct 

### ISSUE THREAD

**Issue by ManjunathJayaram** [SAP-docs/btp-cloud-platform#178]
### Issue description

Procedure

1. Navigate to your subaccount. For more information, see [Navigate in the Cockpit](https://help.sap.com/docs/btp/sap-business-technology-platform/navigate-in-cockpit).
2. Choose Security > Roles Collections in the navigation area and include the roles into the roles collection.
3. Choose Security > Trust Configuration and assign role collections to users.

3rd step in the above procedure is not correct for assigning roles to users.

### Feedback Type (Optional)

None

### Page Title on SAP Help Portal (prefilled)

Configure Application Roles and Assign Roles to Users

### Page URL on SAP Help Portal (prefilled)

https://help.sap.com/docs/btp/sap-business-technology-platform/configure-application-roles-and-assign-roles-to-users

**Comment by danielforsap** [comment]
@NHingerl  this isn't my topic. 

Daniel

**Comment by nikolasimeonov77** [comment]
Thank you for your feedback! We’ll look into it and come back to you if we have any questions.

**Comment by MichaelPShea** [comment]
@ManjunathJayaram : Yes, thanks for the feedback. Steps two and three were originally meant as two separate options for assigning role collections. The user interface has since changed and step three no longer works as described. 

In fact the whole story is better described here: https://help.sap.com/docs/btp/sap-business-technology-platform/building-roles-and-role-collections-for-applications?version=Cloud

I will update the documentation

**Comment by ManjunathJayaram** [comment]
https://people.sap.com/jmy2kor
